unittest/test_lammps.c

```diff
@@ -0,0 +1,86 @@
+#include "utest.h"
+#include <string.h>
+
+#include <md_lammps.h>
+#include <md_trajectory.h>
+#include <md_molecule.h>
+#include <core/md_allocator.h>
+#include <core/md_os.h>
+#include <core/md_log.h>
+
+UTEST(lammps, parse_small) {
+    md_allocator_i* alloc = md_heap_allocator;
+
+    str_t path = STR(MD_UNITTEST_DATA_DIR"/Water_Ethane_Cubic_Init.data");
+    md_lammps_data_t lammps_data = {0};
+    data_format_t *formatPtr, format;
+    formatPtr = &format;
+    get_data_format(formatPtr, full);
+
+    bool result = md_lammps_data_parse_file(&lammps_data, path, alloc, formatPtr);
+    EXPECT_TRUE(result);
+    EXPECT_EQ(lammps_data.num_atoms, 7800);
+    EXPECT_EQ(lammps_data.num_atom_types, 4);
+    EXPECT_EQ(lammps_data.atom_type_mass[0].mass, 1.008f);
+    EXPECT_EQ(lammps_data.bonds[0].second_atom_idx, 2);
+    EXPECT_EQ(lammps_data.cell_def.xy, 0);
+
+    
+
+    md_molecule_t mol = {0};
+
+    md_lammps_molecule_init(&mol, &lammps_data, alloc);
+    for (int64_t i = 0; i < mol.atom.count; ++i) {
+        EXPECT_EQ(mol.atom.x[i], lammps_data.atom_data[i].x);
+        EXPECT_EQ(mol.atom.y[i], lammps_data.atom_data[i].y);
+        EXPECT_EQ(mol.atom.z[i], lammps_data.atom_data[i].z);
+        EXPECT_NE(mol.atom.mass[i], 0);
+        EXPECT_NE(mol.atom.element[i], 0);
+    }
+    md_molecule_free(&mol, alloc);
+
+    /*
+    EXPECT_TRUE(lammps_init_from_file(&mol, path, alloc, formatPtr));
```

Comment:
This can be removed, so I don't have to do it later :)

Reply:
Removed, as well as some other blocks